Objective-C/Tests/URLEndpointListenerTest.m

```diff
@@ -460,11 +462,20 @@ - (void) testMultipleListenersOnSameDatabase {
     
     [listener1 stop];
     [listener2 stop];
+    [self ignoreException:^{
+        NSError* error = nil;
+        Assert([listener1.config.tlsIdentity deleteFromKeyChainWithError: &error],
+               @"Couldn't delete identity1: %@", error);
+        Assert([listener2.config.tlsIdentity deleteFromKeyChainWithError: &error],
+               @"Couldn't delete identity2: %@", error);
+    }];
```

Comment:
This test creates two listeners and starts them manually. The TLSIdentities of the two listeners needs to be cleaned up at the end of the test.

Reply:
do you think anything else to be cleaned up?

Reply:
listener1.config.tlsIdentity and listener2.config.tlsIdentity.

Reply:
line 554 & 556 ?

Reply:
Oh, I didn't see that. Thanks.